### ISSUE TITLE
Make a more accessible/higher contrast contact form

### ISSUE THREAD

**Issue by sbinlondon** [foundersandcoders/techforbetter-website#89]
The current one looks great but isn't so great for accessibility - no focus around the boxes, and you can barely see the colour change when you're in a certain box.

I propose to:

- bring back the focus glow around the boxes, but make it a color that matches the current palette
- have the active/hover colors of the background of the box be more obvious

**Comment by sima-qian** [comment]
With regard to focus in the contact form, the outline is there, just not visible. I hid it on the form because the form input elements have rounded edges and there is no way to give the outline rounded edges.

My solution was to set the distance between the element and the outline to a few pixels, and set the colour to the background colour of the footer - so the outlines are invisible, but still there for accessibility.

The `background-color` of focussed form elements is an RGBA value, so you could turn up the opacity and see how it looks. I personally felt `0.1` looked nicest, and left the white text very readable.
